sdk/core/azure-core-serializer-json-jackson/CHANGELOG.md

```diff
@@ -9,11 +9,12 @@
   `SecurityManager` policies as `SecurityManager` is deprecated for removal. This `JacksonAdapter` will serve as the
   long term replacement to the `JacksonAdapter` in `azure-core`. ([#33401](https://github.com/Azure/azure-sdk-for-java/pull/33401))
 
-### Breaking Changes
+### Other Changes
 
-### Bugs Fixed
+#### Dependency Updates
 
-### Other Changes
+- Upgraded `azure-core` from `1.36.0` to `1.37.0`.
+- Upgraded Jackson to the latest releases `2.1.X`.
```

Comment:
We should specify which versions the Jackson dependencies were updated to, the latest one I can find in the CHANGELOG says `2.13.4`, so `2.1.x` seems misleading